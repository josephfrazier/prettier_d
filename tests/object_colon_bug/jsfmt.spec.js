run_spec(__dirname, ["flow", "typescript"], { printWidth: 35 });
