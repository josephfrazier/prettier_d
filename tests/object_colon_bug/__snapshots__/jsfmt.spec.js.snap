// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bug.js 1`] = `
====================================options=====================================
parsers: ["flow", "typescript"]
printWidth: 35
                                   | printWidth
=====================================input======================================
const foo = {
  bar: props.bar ? props.bar : noop,
  baz: props.baz
}

=====================================output=====================================
const foo = {
  bar: props.bar
    ? props.bar
    : noop,
  baz: props.baz
};

================================================================================
`;
